### ISSUE TITLE
Polylines crossing international dateline don't render in 2D

### ISSUE THREAD

**Issue by Maarondesigns** [CesiumGS/cesium#9052]
Sandcastle example: 
[Fiji to LA](https://sandcastle.cesium.com/#c=jVJRT9swEP4rVp5SKTh1ktZJCNWm0m1ITKANjYdlD6Y5wJtjR7bTqpv63xcTCi2NpvnFvrvv+/TdnVdMoxWHNWh0hiSs0RwMb2v87Snnl97yKZ4raRmXoEtvdFrKUgqw6AP/yQ9Zc6Zt92Iyxvda1efwoAGMT2iKkySLJ9E0HtMoI1FwQiimSTqZ0DihlMY0o07YyV6+/y/RE0JSHJFs2klkCUnGGQ3iGKfpOOpyU0pInJBXs40SG9F1cKi9kJbbjf+nlGjv7LA5elPoi4ZbrqTJ0Xc3gaAz/CM4xi0Fq5sb9VGrVlY5srqFAdSaV/YxR8lAqWYWNGci3zd8/WztqrXu+vyMudaqAX3cyYsZJZTOXybqIvxlcR4Mo1Uvftt7i/6Nmg9I3366uFkcs7ajw9z2Ndz2m+q/Iga3Fg4Gs6ryd9sYnXqBVxi7ETDbEd/xulHaolYLH+PQQt2IbiQmvGuXv8DipTFO2EGLcJ9aVHyFeHU28MHd5ozpKvetEF/5byi9WRF2+COqUKzi8uFqBVqwjYM9ktlln8QYF2EXDjOtUuKO6TfKfwE)

Browser:
Chrome Version 84.0.4147.89 (Official Build) (64-bit)

Operating System: 
Windows 10 Version 1903

**Comment by Maarondesigns** [comment]
Polyline gets cut where the map starts wrapping again. Here you can see the line in the base layer as well.
![image](https://user-images.githubusercontent.com/40376601/88103242-be6ee080-cb6e-11ea-9ca4-297635afb9c1.png)

**Comment by hpinkos** [comment]
Thanks @Maarondesigns.  This looks like it's specifically a problem with `clampToGround: true`.  If you remove that line, the polyline renders correctly.

cc @likangning93

**Comment by Maarondesigns** [comment]
> Thanks @Maarondesigns. This looks like it's specifically a problem with `clampToGround: true`. If you remove that line, the polyline renders correctly.
> 
> cc @likangning93

Ah, got it, thanks. Is there some easy way set clampToGround: false whenever the viewer is in 2D mode, but true when in 3D? I am using this feature in a travel app and it would be pretty tedious to loop through all trips and update each polyline whenever the user switches between 2D and 3D. 
Thanks again!

**Comment by hpinkos** [comment]
@Maarondesigns yeah, I was going to suggest a `CallbackProperty`, but the performance of clamped polylines are too slow for it to be re-computed every frame (which is what the CallbackProperty is making the Entity do behind the scenes).  I don't have any other easy ideas, other than always having `clampToGround: false` in both 2D and 3D.  Not sure what your use case for having it clamped is.

**Comment by Maarondesigns** [comment]
> @Maarondesigns yeah, I was going to suggest a `CallbackProperty`, but the performance of clamped polylines are too slow for it to be re-computed every frame (which is what the CallbackProperty is making the Entity do behind the scenes). I don't have any other easy ideas, other than always having `clampToGround: false` in both 2D and 3D. Not sure what your use case for having it clamped is.

It's just a bonus feature when creating trip markers connected with polylines, and the user can move the markers by clicking and dragging them. I can't take away clamping the lines to the terrain now that the user has already seen it :)
